src/applications/mhv-secure-messaging/tests/components/Compose/ComposeForm.unit.spec.jsx

```diff
@@ -1527,69 +1526,17 @@ describe('Compose form component', () => {
 
   it('should contain Edit Signature Link', () => {
     const customState = { ...initialState, featureToggles: { loading: false } };
-    // eslint-disable-next-line camelcase
     customState.sm.preferences.signature.includeSignature = true;
     const screen = setup(customState, Paths.COMPOSE);
     expect(screen.getByText('Edit signature for all messages')).to.exist;
   });
 
-  it('renders combobox when combobox feauture flag is true', () => {
```

Comment:
no longer using a dedicated toggle for combobox as it will be released as a part of the new curated recipient list flow